packages/react/src/components/MultiSelect/FilterableMultiSelect.js

```diff
@@ -323,6 +323,7 @@ export default class FilterableMultiSelect extends React.Component {
           <Downshift
             {...downshiftProps}
             highlightedIndex={highlightedIndex}
+            id={id}
```

Comment:
Does passing the `id` to Downshift cause any kind of duplicate id issues or do they use it for internal behavior? 👀

Reply:
they use it otherwise they will autogenerate the aria props by default and override any props we have on downshift children, related https://github.com/downshift-js/downshift/issues/602